.eslintrc

```diff
@@ -1,6 +1,5 @@
 {
   "root": true,
-  "parser": "@typescript-eslint/parser",
```

Comment:
just curious about this change, is it not necessary anymore?

Reply:
I don't believe so. I was actually getting errors when adding new .tsx files and my Googling indicated this could be part of the problem. Removing it made all the errors vanish, but I'm willing to try it again as well.

Reply:
So, it may actually just be an issue with VS code. The error I was getting _only_ occurred with new component files. When I reloaded VS code, it seemed to pick up the correct parser: https://github.com/typescript-eslint/typescript-eslint/issues/864

Now that I know the issue, I can work around it. Re-adding parser